charts/stable/minecraft-java/values.yaml

```diff
@@ -2,79 +2,75 @@ image:
   pullPolicy: Always
 j17j9Image:
   repository: tccr.io/truecharts/minecraft-java17-openj9
-  tag: v2023.4.1@sha256:81e86ae5691d594207a9aaabbca34efacb8887e50766b4872ce3d769df3e6eb5
+  tag: v2023.4.1@sha256:a0c78f26955bdd19117db78f008749345af9fbdce7be4724efddeb4e1aea4495
   pullPolicy: Always
 j17jdkImage:
   repository: tccr.io/truecharts/minecraft-java17-jdk
-  tag: v2023.4.1@sha256:281daf069cecacf710e2f3b7b8bdb4882e96bf655605204b2f33f259b5f8cbc9
+  tag: v2023.4.1@sha256:b0e951b55d1b91a1bb6a6ae7b7585526b60d95bd34e11281de7e701197b1d34c
   pullPolicy: Always
 j17gceImage:
   repository: tccr.io/truecharts/minecraft-java17-graalvm-ce
-  tag: latest@sha256:57e90c3b98ff367cb3491c594d95e8cc89cd677f1168430885d3e739c77f7f2f
+  tag: latest@sha256:20e8f1dd890f9ef49a58e865039496a13dd9e46dde723f335cf037122b83a0b2
   pullPolicy: Always
 j17alpineImage:
   repository: tccr.io/truecharts/minecraft-java17-alpine
-  tag: v2023.4.1@sha256:5486bdee5464ffdeead25d1bf7072f935313e857a0406524f80e67f1e1cd1543
+  tag: v2023.4.1@sha256:f89317c45ae57345d756b5711d348288141248a01de6f9d8b3ba8893ffb2fc59
   pullPolicy: Always
 j11Image:
   repository: tccr.io/truecharts/minecraft-java11
   tag: latest@sha256:de15c958a648cad5a0813a528e36ab8e78e84f95e572c291aa3e4de36aae5556
   pullPolicy: Always
 j11jdkImage:
   repository: tccr.io/truecharts/minecraft-java11-jdk
-  tag: v2023.4.1@sha256:f899603d8a7913e9e18c23dc8b63a0716872c176b11faaaf69af702636b02dc0
+  tag: v2023.4.1@sha256:b0e951b55d1b91a1bb6a6ae7b7585526b60d95bd34e11281de7e701197b1d34c
   pullPolicy: Always
 j11j9Image:
   repository: tccr.io/truecharts/minecraft-java11-openj9
-  tag: v2023.4.1@sha256:ec93725bc2ca1ee942fc5741cd1ca22e9472d2771dae7b399873b52a795a5ab1
+  tag: v2023.4.1@sha256:1d15fe6df28051934505a0eb52331dfac63dd506bcc9d6f7179a58184aed5028
   pullPolicy: Always
 j8Image:
   repository: tccr.io/truecharts/minecraft-java8
-  tag: latest@sha256:7c70936fa9029de588ead87bc44f554c53371aa75763f617a1d55da46c4a09db
+  tag: latest@sha256:59b850aed3a9507932bd8266c8d0d5ff3a3baecb94c43ebec931d6871c497353
   pullPolicy: Always
 j8gceImage:
   repository: tccr.io/truecharts/minecraft-java8-graalvm-ce
-  tag: latest@sha256:0c39ac645c01411997b98ff41bdf8cc0495d92b2b6dbe0b7f9129ba356807ceb
+  tag: latest@sha256:7564fd2bb91bc84bfa2bfeb2d3f4331329066771716d2d03482a57774ceb3afa
   pullPolicy: Always
 j8jdkImage:
   repository: tccr.io/truecharts/minecraft-java8-jdk
-  tag: v2023.4.1@sha256:f50331c469d574c7b2bf0d481f4a6a9f4e155baf07c4e531d94e7f178e4d7ecf
+  tag: v2023.4.1@sha256:f9d8a8f43b98977eecd72d234e10d943c8e567ff10aefd425476ddc2633d43aa
   pullPolicy: Always
 j8j9Image:
   repository: tccr.io/truecharts/minecraft-java8-openj9
-  tag: v2023.4.0@sha256:23ef7a412217aca78065650aca969868d6411f252dda7a5ff6feba3237340d67
+  tag: v2023.4.1@sha256:8ef5e9d3e25d32a541f89d616289003f5cee8b15cb0569e79b2e778d3b0b68d4
   pullPolicy: Always
+
+tty: true
+stdin: true
```

Comment:
Not here
https://truecharts.org/manual/helm/common/container/

Reply:
we dont need it due to the image having rcon-cli built in the main image.